### PR TITLE
Add Satellite Prod Loading

### LINKED ISSUE #55: Load Satellite data
- add satellite to config - [link](https://github.com/openclimatefix/nowcasting_forecast/tree/main/nowcasting_forecast/config). Good to make new version mvp_v2.yaml
- add fake data fixture, and test on new config

have to adjust nowcast_dataset - for satellite. Load netcdf file. 
See NWP for [example](https://github.com/openclimatefix/nowcasting_dataset/blob/main/nowcasting_dataset/data_sources/nwp/nwp_data_source.py#L195)

### PR DESCRIPTION
**Pull request by jacobbieker**
# Pull Request

## Description

Ads loading the HRV and non-HRV production files

Fixes #55 

## How Has This Been Tested?

Unit tests

- [x] Yes

_If your changes affect data processing, have you plotted any changes? i.e. have you done a quick sanity check?_

- [ ] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/nowcasting/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by peterdudfield** [comment]
Sorry about the conflicts, I'm hapy to help conflict resolve if you like?

**Comment by jacobbieker** [comment]
> Sorry about the conflicts, I'm hapy to help conflict resolve if you like?

I can get the merge conflicts, thanks though!

File: tests/conftest.py [hunk 1]
```diff
@@ -185,7 +185,7 @@ def pv_yields_and_systems(db_session):
             ).to_orm()
             if datetime_utc.hour in [22, 23, 0, 1, 2]:
                 pv_yield_1.solar_generation_kw = 0
-            pv_yield_1.pv_system = pv_system_sql_1
+            pv_yield_1.pv_system = pv_system_sql_1s
```
Comment (peterdudfield) [inline tests/conftest.py:188]: change this back?

Comment (jacobbieker) [reply tests/conftest.py:188]: Ah okay, yeah, I was wondering why tests were failing on the PV systems, thanks!

**Comment by peterdudfield** [comment]
Just so I remeber, current area is due to KeyError: 'area' in satellite

[code](https://github.com/openclimatefix/nowcasting_dataset/blob/main/nowcasting_dataset/data_sources/satellite/satellite_data_source.py#L87)

**Comment by jacobbieker** [comment]
Okay, @peterdudfield I think its ready, I was getting errors on the random image coordinates, so also copied over the coordinates from the live data for the fake data too.

File: tests/test_batch.py [hunk 2]
```diff
@@ -31,13 +32,21 @@ def test_make_batches_mvp_v1(nwp_data, pv_yields_and_systems):
...
+        with zarr.ZipStore(hrv_sat_path) as store:
+            hrv_sat_data.to_zarr(store, compute=True)
+        os.environ["HRV_SAT_PATH"] = hrv_sat_path
+        sat_path = f"{temp_dir}/sat_unittest.zarr.zip"
+        with zarr.ZipStore(sat_path) as store:
+            sat_data.to_zarr(store, compute=True)
+        os.environ["SAT_PATH"] = sat_path
 
         make_batches(
             config_filename="nowcasting_forecast/config/mvp_v2.yaml", temporary_dir=temp_dir
```
Comment (peterdudfield) [inline tests/test_batch.py:52]: small thing below here
Could you check that satellite and hrv batches have been made - just to be double sure

**Review by peterdudfield — APPROVED** [review]
Looks great - well done for this

Would you be able to add that extra assert (see comment) and then its good to go

File: tests/test_batch.py (hunk 2 above)
Comment (jacobbieker) [reply tests/test_batch.py:52]: Okay, sure

**Pull request merged**